astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -56,188 +40,367 @@ The login flow will look like this after FusionAuth is introduced.
-docker-compose up -d
 ```
 
-The FusionAuth configuration files also make use of a unique feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart): when FusionAuth comes up for the first time, it will look at the Kickstart file located at `kickstart/kickstart.json` and mimic API calls to configure FusionAuth for use when it is first run.
+### Run FusionAuth Via Docker
+
+You'll find a Docker Compose file `docker-compose.yml` and an environment variables configuration file `.env` in the root directory of the repo.
+
+Assuming you have Docker installed, you can start FusionAuth on your machine with the following.
+
+```shell
+docker compose up -d
+```
+
+This will start three containers, one each for FusionAuth, Postgres, and Elasticsearch.
 
-<Aside type="note" >If you ever want to reset the FusionAuth system, delete the volumes created by docker-compose by executing `docker-compose down -v`.</Aside>
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart). When FusionAuth starts for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to your specified state.
 
-FusionAuth will be initially configured with these settings:
+<Aside type="note">If you ever want to reset the FusionAuth system, delete the volumes created by `docker compose` by executing `docker compose down -v`, then rerun `docker compose up -d`.</Aside>
 
-* Your client Id is: `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`
-* Your client secret is: `super-secret-secret-that-should-be-regenerated-for-production`
+FusionAuth will be configured with these settings:
+
+* Your client Id is `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is `super-secret-secret-that-should-be-regenerated-for-production`.
 * Your example username is `richard@example.com` and your password is `password`.
 * Your admin username is `admin@example.com` and your password is `password`.
-* Your fusionAuthBaseUrl is 'http://localhost:9011/'
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.
 
-You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
 
-## Create your Spring Boot Application with Thymeleaf
+### Create A Basic Spring Boot Application
 
-### Overview of the Application
+Next, you'll set up a basic Spring Boot project with a single app. While this guide builds a new Spring Boot project, you can use the same method to integrate your existing project with FusionAuth.
 
```

Comment:
Adding this because we have it in other quick starts.  The comments in git through this off so please make sure the "</Aside>" tag is included.

```suggestion

<Aside type="note">
If you simply want to run the application , there is a completed version in the 'complete-application' directory. You can use the following commands to get it up and running if you do not want to create your own.

```shell
cd complete-application
./mvnw package
./mvnw spring-boot:run
```

Then view the website at the following location `http://localhost:8080`.
</Aside>
```